.pipelines/deploy-azure-env.yml

```
@@ -7,6 +7,13 @@ parameters:
 - name: vsoConfigBuildID
 - name: vsoDeployerBuildID
 
+resources:
```

Uses `AROGIT` service connection for readonly access to msazure organization to pull the config from `AzureRedHatOpenShift/RP-Config` repository.

---

When selecting the tagged release config the resource would look like
```suggestion
resources:
  repositories:
  - repository: rpconfig
    type: git
    endpoint: AROGIT
    name: AzureRedHatOpenShift/RP-Config
    ref: refs/tags/v2020.01.02.01
```

Which translates to 
```suggestion
parameters:
- name: location
- name: vsoConfigBuildID
- name: vsoDeployerBuildID
- name: version

resources:
  repositories:
  - repository: rpconfig
    type: git
    endpoint: AROGIT
    name: AzureRedHatOpenShift/RP-Config
    ref: refs/tags/${{ parameters.version }}
```

to make it configurable when starting the pipeline